analyses/cell-type-nonETP-ALL-03/scripts/writeout_submission.R

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env Rscript
+
+library(Seurat)
+library(ggplot2)
+
+writeout <- function(ind.lib, ct.colors = ct_color, project.ID = projectID, n.row = 1){
+  seu <- readRDS(file.path(out_loc,"results/rds",paste0(ind.lib,".rds")))
+  voi <- c('newB.copykat.pred','sctype_classification')
+  changeName.voi <- c('tumor_cell_classification','cell_type_assignment')
```

Comment:
Thank you for the kind reminder! Is it okay, if I use "tumor", "normal", or "unknown", because there are some cells labeled as "not.defined" from CopyKat?

Reply:
Yes, including "unknown" sounds good.